tools/aquainfra_importer/aquainfra_importer.xml

```diff
@@ -0,0 +1,13 @@
+<tool id="aquainfra_importer" name="AquaINFRA Importer" tool_type="data_source" version="1.0" profile="22.05">
+    <description>downloads content via the AquaINFRA interaction platform</description>
+    <command><![CDATA[
+        python '$__tool_directory__/data_source.py' '$output' $__app__.config.output_size_limit '$output'
```

Comment:
When installing from the toolshed, this will only work if you also provide the `data_source.py` script together with the .xml because this literally expects the .xml and the .py file in the same directory.
If you haven't customized anything you can just copy over https://github.com/galaxyproject/galaxy/blob/dev/tools/data_source/data_source.py here.

Reply:
but the aquainfra part of this is not implemented yet? or should it be?

Reply:
Yeah, I wasn't sure what to do first. The problem is that only very very few datasets have metadata including a direct download link. Most don't have a link or redirect to the website of the data provider where you need to accept conditions or login or similar. Ideally the "Import to Galaxy" Button would be on the website of the data provider but I don't see that happen in the near future. 
Give me some time to update the platform and I will show you an example record to test the import to Galaxy.